Appraisals

```diff
@@ -47,15 +47,12 @@ end
 appraise "rails6.0" do
   gem "byebug"
   gem "capybara", ">= 2.15"
-  gem "jbuilder", "~> 2.7"
```

Comment:
Let's remove these gems from the other versions as well, if possible. We might speed things up a little for those test runs as well.

Reply:
Let's also see if we can get rid of capybara, selenium-webdriver, and web-console, which shouldn't be needed anymore since we are in api mode.